threadfin/reconcile/reconcile.py

```diff
@@ -222,9 +222,10 @@ def write_web_page(self, fname, date):
         with open(fname, 'w') as fh:
             fh.write(self.web_page(date))
 
-
+OTS_BOOKKEEPING_DIR = os.getenv("OTS_BOOKKEEPING_DIR" default = ".")
+    
 @click.command()
-@click.option('--config', default="config.yaml", help="config file")
+@click.option('--config', default=os.path.join(OTS_BOOKKEEPING_DIR, "threadfin/examples/config_samples/setupsample.yaml"), help="config file")
```

Comment:
I think it might be better to require that the user pass a `--config` option, since in actual use they will always have to.  Having the program's invocation method be different for the sample data than for real data partially defeats the purpose of having sample data in the first place :-).

The way to tell the user how to try things out with sample data is to document it in the README, or wherever the usage instructions are.  So, relatedly to my comment above default behavior, this PR needs to include any appropriate documentation updates to match the code changes you've made (and that's a rule in general for PRs).